README.md

```diff
@@ -1,31 +1,48 @@
 # Document Understanding Solution
 
-DUS leverages the power of Amazon Textract, Amazon Comprehend , Amazon Comprehend Medical and Amazon Elasticsearch to provide digitization , domain-specific data discovery, redaction controls , structural component extraction and other document processing & understanding capaibilities.
+DUS leverages the power of Amazon Textract, Amazon Comprehend , Amazon Comprehend Medical Amazon Elasticsearch and Amazon Kendra to provide digitization , domain-specific data discovery, redaction controls , structural component extraction and other document processing & understanding capaibilities.
 
 Current document formats supported : **PDF,JPG,PNG**
 
 The instructions below cover installation on Unix-based Operating systems like macOS and Linux. 
 You can use a AWS Cloud9 enviornment or EC2 instance (recommended: t3.large or higher on Amazon Linux platform) to deploy the solution
-To run the solution,first clone the project
+To run the solution,first clone the project and decide which region you would want to deploy the solution.
+*Note* : The AWS services used in this solution are not all available in all AWS Regions. Please refer the [AWS Regions Table](https://aws.amazon.com/about-aws/global-infrastructure/regional-product-services/) and check where the services in DUS are available for choosing the deployment region region
```

Comment:
region is appearing twice. Should be there only once.

Reply:
Updated. Thanks!